perceval/components/core_catalog/gates_1qubit.py

```diff
@@ -0,0 +1,200 @@
+Q1:(dual rail)──┤  H  ├── (dual rail) :Q1
+              ──┤     ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('h')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        return Circuit(2, name="H") // BS.H()
+
+
+class PauliXItem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤Pauli├── (dual rail) :Q1
+              ──┤  X  ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('x')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        return Circuit(2, name='X') // PERM([1,0])
+
+
+class RxItem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤ Rx  ├── (dual rail) :Q1
+              ──┤     ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('rx')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        theta = kwargs.get('theta', 0.0)
+        return Circuit(2, name=f"Rx({theta:.3})") // BS.Rx(theta=-theta)
+
+
+class PauliYItem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤Pauli├── (dual rail) :Q1
+              ──┤  Y  ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('y')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        return Circuit(2, name="Y") // PERM([1, 0]) // (1, PS(math.pi / 2)) // (0, PS(-math.pi / 2))
+
+
+class RyItem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤ Ry  ├── (dual rail) :Q1
+              ──┤     ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('ry')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        theta = kwargs.get('theta', 0.0)
+        return Circuit(2, name=f"Ry({theta:.3})") // BS.Ry(theta=theta)
+
+
+class PauliZItem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤Pauli├── (dual rail) :Q1
+              ──┤  Z  ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('z')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        return Circuit(2, name="Z") // (1, PS(-math.pi))
+
+
+class RzItem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤ Rz  ├── (dual rail) :Q1
+              ──┤     ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('rz')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        theta = kwargs.get('theta', 0.0)
+        return Circuit(2, name=f"Rz({theta:.3})") // (0, PS(-theta / 2)) // (1, PS(theta / 2))
+
+
+class PhaseShiftITem(ASingleQubitGate):
+    str_repr = r"""                ╭─────╮
+Q1:(dual rail)──┤ PS  ├── (dual rail) :Q1
+              ──┤     ├──
+                ╰─────╯ """
+
+    def __init__(self):
+        super().__init__('ph')
+
+    def build_circuit(self, **kwargs) -> Circuit:
+        phi = kwargs.get('phi')
+        return Circuit(2, name="phase shift") // (1, PS(phi))
```

Comment:
Write the angle in the name of the circuit like Rx, Ry and Rz. Also, the name can probably be shortened